prow/cluster/jobs/istio/api/istio.api.master.gen.yaml

```diff
@@ -172,6 +204,14 @@ presubmits:
     decorate: true
     name: gencheck_api
     path_alias: istio.io/api
+    reporter_config:
```

Comment:
I think this is a presubmit test? So it should not trigger slack

Reply:
good catch
/hold